gef.py

```diff
@@ -2867,14 +2865,34 @@ def get_terminal_size():
     """Return the current terminal size."""
     if is_debug():
         return 600, 100
+    if platform.system() == "Windows":
+        from ctypes import windll, create_string_buffer
+        #stdin handle = -10
+        #stdout handle = -11
+        #stderr handle = -12
+        herr = windll.kernel32.GetStdHandle(-12)
+        csbi = create_string_buffer(22)
+        res = windll.kernel32.GetConsoleScreenBufferInfo(herr, csbi)
+        if res:
+            (_bufx, _bufy, _curx, _cury, _wattr,
```

Comment:
The variables `_bufx`, `_bufy`, etc. are not used in the scope, might as well remove them with `_`
```python
_,_,_,_,_, left, top, right, bottom, _, _ = struct.unpack("hhhhHhhhhhh", csbi.raw)
```
is more readable

Reply:
HI
I added these changes in the new commit.